### PR TITLE
Fixed #34234 -- Dropped support for PROJ 4.

### PR DESCRIPTION
**Pull request by select-case**

### CODE REVIEW

**Review by claudep — CHANGES_REQUESTED** [review]
Thanks! You should also add a note in the release notes. I think we could also simplify a bit in `tests/gis_tests/test_spatialrefsys.py` (near the top of the file).

File: docs/releases/4.2.txt [hunk 1]
```diff
@@ -531,9 +531,12 @@ Miscellaneous
 
 * The minimum supported version of `redis-py`_ is increased from 3.0.0 to
   3.4.0.
-
+
```
Comment (felixxm) [inline docs/releases/4.2.txt:534]: Please revert this unrelated change.

File: tests/gis_tests/test_spatialrefsys.py [hunk 2]
```diff
@@ -11,7 +11,6 @@
         "auth_srid": 4326,
         # Only the beginning, because there are differences depending on installed libs
         "srtext": 'GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84"',
-        # +ellps=WGS84 has been removed in the 4326 proj string in proj-4.8
```
Comment (felixxm) [inline tests/gis_tests/test_spatialrefsys.py:14]: It's not only about the comment, we should update `proj_re` and related tests.

**Comment by felixxm** [comment]
@select-case Thanks :+1: I pushed edits.

**Review by claudep — APPROVED** [review]
:+1:

**Comment by felixxm** [comment]
@claudep Thanks for checking :+1:

**Pull request merged**